composer.json

```diff
@@ -3,12 +3,20 @@
         {
             "type":"composer",
             "url":"https://wpackagist.org"
+        },
+        {
+            "type":"vcs",
+            "url":"https://github.com/mitlibraries/mitlibraries-parent"
```

Comment:
I think this is useful for deploying, but not for local development as it won't leave it as an independent git repo. The process described in the readme is clunky, but leaves us with independent version controlled themes / plugins which may be better for our local dev needs.

Reply:
Hmm...good point. I had thought that this allowed for local development but I see that it doesn't show up as a git repo by the point this shows up. Let me do some more digging...